apps/chat/.env.production

```
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_PWA_URL=https://pwa.klicker.uzh.ch
+NEXT_PUBLIC_CHAT_URL=https://chat.klicker.uzh.ch
+
+MCP_URL=TODO
+MCP_KEY=TODO
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)